linera-execution/src/lib.rs

```diff
@@ -1082,10 +1094,30 @@ impl ExecutionRuntimeContext for TestExecutionRuntimeContext {
             admin_chain_id: dummy_chain_description(0).id(),
             genesis_config_hash: CryptoHash::test_hash("genesis config"),
             genesis_timestamp: Timestamp::from(0),
+            genesis_committee_blob_hash: CryptoHash::test_hash("genesis committee"),
             name: "dummy network description".to_string(),
         }))
     }
 
+    async fn committees_for(
+        &self,
+        epochs: BTreeSet<Epoch>,
+    ) -> Result<BTreeMap<Epoch, Committee>, ViewError> {
+        let committee_blob_bytes = self
+            .blobs
+            .iter()
+            .find(|item| item.key().blob_type == BlobType::Committee)
+            .ok_or_else(|| ViewError::NotFound("committee not found".to_owned()))?
+            .value()
+            .bytes()
+            .to_vec();
+        let committee: Committee = bcs::from_bytes(&committee_blob_bytes)?;
+        Ok(epochs
+            .into_iter()
+            .map(|epoch| (epoch, committee.clone()))
+            .collect())
```

Comment:
That may be fine for all the tests we currently have, but it's really just assigning the first committee blob it found to all epochs. Let's at least add a `TODO` and create an issue to fix this.